testing/integration-neo4j/src/test/java/com/neueda/jetbrains/plugin/graphdb/test/integration/neo4j/tests/cypher/inspection/CypherExplainWarningInspectionTest.java

```diff
@@ -0,0 +1,60 @@
+package com.neueda.jetbrains.plugin.graphdb.test.integration.neo4j.tests.cypher.inspection;
+
+import com.intellij.psi.PsiFile;
+import com.neueda.jetbrains.plugin.graphdb.jetbrains.component.datasource.state.DataSourceApi;
+import com.neueda.jetbrains.plugin.graphdb.jetbrains.inspection.CypherExplainWarningInspection;
+import com.neueda.jetbrains.plugin.graphdb.jetbrains.util.NameUtil;
+import com.neueda.jetbrains.plugin.graphdb.test.integration.neo4j.util.base.BaseIntegrationTest;
+
+public class CypherExplainWarningInspectionTest extends BaseIntegrationTest {
+
+    private DataSourceApi dsApi;
+
+    @Override
+    public void setUp() throws Exception {
+        super.setUp();
+        this.dsApi = dataSource().neo4j31();
+        myFixture.enableInspections(CypherExplainWarningInspection.class);
+    }
+
+    public void testNonDataSourceFile_NoHighlight() {
+        PsiFile psiFile = myFixture.addFileToProject("a.cyp", "MATCH (a)-->(b) RETURN *");
+        myFixture.configureFromExistingVirtualFile(psiFile.getVirtualFile());
+        myFixture.checkHighlighting();
+    }
+
```

Comment:
There should be test that check behaviour if we are in data source bound file, but there are no actual data source,